injected/entry-points/windows.js

```diff
@@ -14,9 +14,7 @@ function initCode() {
     const userPreferences = $USER_PREFERENCES$;
 
     const processedConfig = processConfig(config, userUnprotectedDomains, userPreferences, platformSpecificFeatures);
```

Comment:
I'm not really sure why we import something from utils to then pass it back in again... but it's also fine not to change this.

In Deeps change, we've tried to move more stuff out of these boilerplate files when it's not needed. I think we can do much more of it and/or almost get rid of them now.

Reply:
yeah I can change this tbh

Reply:
Ahh I remember now, it's to allow things like autofill to call `processConfig`. It's not ideal and we can continue to tweak, but as you mentioned let's not alter it as part of this change